Lib/ufo2ft/featureWriters/kernFeatureWriter.py

```diff
@@ -424,10 +424,16 @@ def _makeKerningLookups(self):
         if shouldSplit:
             # make one DFLT lookup with script-agnostic characters, and two
             # LTR/RTL lookups excluding pairs from the opposite group.
-            # We drop kerning pairs with ambiguous direction.
+            # We drop kerning pairs with ambiguous direction: i.e. those containing
+            # glyphs from scripts with different overall horizontal direction, or
+            # glyphs with incompatible bidirectional type (arabic numbers vs
+            # arabic numerals).
```

Comment:
What is the difference between numbers and numerals?

Reply:
oops, I didn't mean that. I meant "arabic letters vs arabic numerals", i'll fix that